docs/userguide/modules/creating-service-bindings/pages/binding-options.adoc

```diff
@@ -182,4 +182,59 @@ spec:
 ----
 <1>  Database service
 <2> Optional "id" field
-<3>  Translation service
\ No newline at end of file
+<3>  Translation service
+
+== Binding workloads from a label selector
+
+Sometimes, it may be useful to specify the workload being bound by using a label selector.  For
+instance, you may want to bind a service to every workload with the label `environment: production`
+set.  SBO is able to support this kind of binding by being able to bind with label selectors.  As an
+example, to project a single secret into multiple workloads, set up your application field like so:
```

Comment:
Replace L187 to L192 with this:
== Binding workloads using a label selector
You can use a label selector to specify the workload that is being bound. If you declare a service binding using these label selectors to pick up workloads, the {servicebinding-title} periodically attempts to find and bind new workloads that match the given label selector.

For example, you may want to bind a service to every application with the `environment: production` label set in the `application` field. Doing so projects a single secret into multiple workloads.

.Example of `ServiceBinding` CR in the `binding.operators.coreos.com/v1alpha1` API: